.dockerignore

```diff
@@ -0,0 +1,26 @@
+# Build output:
+/.cache/
+/dist/
+/lib/
+/.nyc_output
+/coverage
+
+# Config
+/config.json
```

Comment:
Don't ignore the config file since you actually do want to copy the file into the docker image.

Reply:
config.json is generated by `yarn setup` command on the sync-server.